portfolio/src/main/java/com/google/sps/servlets/Comments.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+public final class Comments {
+
+    private final long id;
+    private final String name;
+    private final String email;
+    private final String message;
+    private final long timeStamp;
+
+    public Comments(long id, String name, String email, String message, long timeStamp) {
+        this.id = id;
+        this.name = name;
+        this.email = email;
+        this.message = message;
+        this.timeStamp = timeStamp;
+    }
+
+    public String getMessage() {
+        return message;
```

Comment:
when accessing attribute inside class, using "this". So this is "return this.message".